sporkfish/searcher/searcher.py

```diff
@@ -52,13 +51,21 @@ def _log_info(
             "depth": depth,
             # time in ms
             "time": int(1000 * elapsed),
-            "nodes": self._statistics.nodes_visited,
-            "nps": int(self._statistics.nodes_visited / elapsed) if elapsed > 0 else 0,
             "score cp": int(score)
             if score not in {float("inf"), -float("inf")}
             else float("nan"),
             "pv": move,  # Incorrect but will do for now
         }
+        total = 0
+        for type in NodeTypes:
```

Comment:
For future, perhaps this deserves to be part of the Stats API?

Reply:
by "this" do you mean the `NodeTypes` enum?

Reply:
I think the logic to print a the stats can live in the API, maybe even as __str__.